packages/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -479,24 +482,37 @@ void main() {
 
       isPaused = await controller.evaluateJavascript('isPaused();');
       expect(isPaused, _webviewBool(false));
-    }, skip: true /* https://github.com/flutter/flutter/issues/72572 */);
+    });
 
     testWidgets('Video plays inline when allowsInlineMediaPlayback is true',
         (WidgetTester tester) async {
       Completer<WebViewController> controllerCompleter =
           Completer<WebViewController>();
       Completer<void> pageLoaded = Completer<void>();
+      // String message1;
```

Comment:
What does this mean (here and below)?

Reply:
oops, some testing code not cleaned up :) Good catch! Thanks